lib/src/room.dart

```diff
@@ -307,6 +310,9 @@ class KurentoRoom extends ChangeNotifier implements Room {
   bool get isSensorDataCollectionEnabled =>
       _serviceRequest.motionSensorDataCollectionEnabled;
 
+  @override
+  String? get ringBackFile => _serviceRequest.ringbackUrl;
```

Comment:
Is there a reason we don't just call this variable `ringbackUrl`? It's not a file, is it?

Reply:
It's MP3, but I'll rename it

Reply:
I meant, it's a url pointing to a mp3, not the actual mp3 file content, right? Usually we name variables "file" when it's the actual binary file. In this case, it's still a string holding a url. Unless I'm mistaken on the content of the value.